pkg/extensions/extension_metrics.go

```diff
@@ -26,10 +26,8 @@ func EnableMetricsExtension(config *config.Config, log log.Logger, rootDir strin
 }
 
 func SetupMetricsRoutes(config *config.Config, router *mux.Router, storeController storage.StoreController,
-	l log.Logger,
+	log log.Logger,
 ) {
-	// fork a new zerolog child to avoid data race
-	log := log.Logger{Logger: l.With().Caller().Timestamp().Logger()}
```

Comment:
Just make sure the original logger has all these options set up.

Reply:
The original logger has those options. All loggers created with NewLogger are created like this:
```
Logger{Logger: log.Hook(goroutineHook{}).With().Caller().Timestamp().Logger()}
```